### DISCUSSION TITLE
유니콘 HTTPS 처럼 KT 인터넷에서 DPI 차단 우회 기능

### DISCUSSION THREAD

**Discussion by piquark6046** [List-KR/community#1] [General]
https://github.com/AdguardTeam/CoreLibs/issues/1789

기존에는 SKT, LG U+에서 작동했지만, KT에서도 작동할 수 있도록 첫 조사가 시작되었습니다.

**Comment by swbaek81** [comment]
윈도우 애드가드프로 환경에서는 혹시 우회방법 있는지 궁금합니다

**Reply by piquark6046** [reply]
SKT와 LG U+에서는 당장 가능은 한데 KT는 nightly에서도 추가가 되지 않은 것으로 기억합니다.

**Reply by swbaek81** [reply]
윈도우10 데스크탑PC 환경이 궁금합니다

**Reply by piquark6046** [reply]
> 윈도우10 데스크탑PC 환경이 궁금합니다

무슨 말씀이신지 모르겠습니다.
AdGuard for Windows 관련된 질문인가요?

**Reply by swbaek81** [reply]
네 윈도우 환경에서 adguard 를 사용하면 우회할수 있는 기능이 혹시 있는지 해서요 adguard vpn 을 쓰는 방법 말고는 없더라고요 lamu.city 사이트요

**Reply by piquark6046** [reply]
KT외 통신사를 사용하시면 `스텔스 모드` -> 최하단에 있는 `DPI로 부터 보호`를 활성화하시면 됩니다.

**Reply by swbaek81** [reply]
오 감사합니다 잘 되네요
